docs/wiki/deployment/configuration.md

```diff
@@ -395,6 +395,22 @@ Example:
 
 There is a strict relationship between the top-level `file_paths` key, and `yara`'s equivalent subkey.
 
+### Prometheus
+
+The `prometheus_targets` key can be used to configure Prometheus targets to be queried. The metric timestamp is taken when the target response is received.  The `prometheus_targets` parent key consists of one child key `urls` of which value contains a list target urls to be scraped.
```

Comment:
hmmm..the MS thing is a standard Prometheus thing, but i guess it wouldn't hurt. Will change.